staker-rewards/rewards.js

```diff
@@ -49,7 +57,36 @@ async function run() {
 
     const operatorsRewards = await calculateOperatorsRewards(context, interval)
 
-    console.table(operatorsRewards)
+    if (process.env.OUTPUT_MODE === "text") {
+      const format = {
+        groupSeparator: "",
+        decimalSeparator: ".",
+      }
+
+      operatorsRewards.forEach((operatorRewards) =>
+        console.log(
+          `${operatorRewards.operator}
+           ${operatorRewards.isFraudulent} 
+           ${operatorRewards.keygenCount}
+           ${operatorRewards.keygenFailCount} 
+           ${operatorRewards.keygenSLA} 
+           ${operatorRewards.signatureCount} 
+           ${operatorRewards.signatureFailCount}
+           ${operatorRewards.signatureSLA} 
+           ${operatorRewards.keepStaked.toFormat(format)} 
+           ${operatorRewards.ethBonded.toFormat(format)} 
+           ${operatorRewards.ethUnbonded.toFormat(format)}
+           ${operatorRewards.ethTotal.toFormat(format)} 
+           ${operatorRewards.ethScore.toFormat(decimalPlaces, format)} 
+           ${operatorRewards.boost.toFormat(decimalPlaces, format)} 
+           ${operatorRewards.rewardWeight.toFormat(decimalPlaces, format)} 
+           ${operatorRewards.totalRewards.toFormat(0, format)}
```

Comment:
`ethScore` and `rewardWeight` should also be displayed without decimals. I propose to do something like:
```
      const decimalPlaces = 2
      const noDecimalPlaces = 0
operatorsRewards.forEach((operatorRewards) =>
        console.log(
          `${operatorRewards.operator}
           // (...)
           ${operatorRewards.ethScore.toFormat(noDecimalPlaces, format)} 
           ${operatorRewards.boost.toFormat(decimalPlaces, format)} 
           ${operatorRewards.rewardWeight.toFormat(noDecimalPlaces, format)} 
           ${operatorRewards.totalRewards.toFormat(noDecimalPlaces, format)}
```

And then move `const decimalPlaces = 2` from the top of the file to `function shortenSummaryValues(summary)`